### ISSUE TITLE
NULL LISTs can be invalid

### ISSUE THREAD

**Issue by hawkfish** [duckdb/duckdb#1651]

**Comment by hawkfish** [comment]
Looks like it is in the `CASE` statement. In File: [execute_case.cpp](https://github.com/duckdb/duckdb/blob/f7b79fdd6e010564860aa64758b907d046b93b16/src/execution/expression_executor/execute_case.cpp#L152); Line 152 we see:


```cpp
		// FIXME the nullmask here is likely borked
		// TODO uuugly
```

**Comment by hawkfish** [comment]
Ironically, the call to `Verify` was incorrect...

**Comment by hawkfish** [comment]
This has been fixed as part of the slice work in #1560
